### DISCUSSION TITLE
Scope of MVP

### DISCUSSION THREAD

**Discussion by christophsax** [BristolMyersSquibb/blockr#125] [General]
For tomorrow's discussion, we could align on the following two questions:
1. What features do we want to be in the MVP? (45 Minutes)
2. How do we organize these features in packages (15 Minutes)

### Which Blocks do we need for MVP?

  - Basic Transformation
    - mutate
    - summarize
    - select
    - arrange
    - filter
  - Basic Plots
  - Pharma Blocks
    - admiral
    - admiral Example given by Karma

### What other Features do we need?

  - Workspace
    - Serializable
    - Storable

  - Extendability
    - BMS developer should be able to develop their own, (e.g. admiral) blocks.
    - With documentation
  

### How do we organize these features in packages?

The solution here should be 'good enough' for the MVP, we can still change it afterwards.

  - blockr 
    - Basics + Transformation, Basic Plots
  - blockr.pharma?
  - Superframework: blockr.bms?

**Comment by christophsax** [comment]
Agenda proposal for today's meeting, @JohnCoene @nbenn @DivadNojnarg 

Please adjust if you like!

**Comment by JohnCoene** [comment]
Could we just have 5 minutes on the serialisation/restoration of dashboards and the workspace object?
